platform/viewer/src/components/Header/Header.js

```diff
@@ -51,26 +54,29 @@ class Header extends Component {
   }
 
   loadOptions() {
-    const { t } = this.props;
+    const {
+      t,
+      user,
+      userManager,
+      modalContext: { show },
+    } = this.props;
     this.options = [
       {
         title: t('About'),
         icon: { name: 'info' },
-        onClick: () => {
-          this.setState({
-            isOpen: true,
-          });
-        },
+        onClick: () =>
+          show(AboutModal, {
+            className: 'AboutModal modal fade themed in',
```

Comment:
Maybe default classes on ModalProvider for fade, themed, etc?
Not sure why AboutModal should care about these, unless we decide to toggle behavior with props

Reply:
Done! thx. Now im not passing className, im using a static property to extract the className from the modal content.